core/src/main/java/org/bptlab/cepta/operators/CountOfTrainsAtStationFunction.java

```diff
@@ -0,0 +1,56 @@
+package org.bptlab.cepta.operators;
+
+import org.apache.flink.streaming.api.datastream.DataStream;
+import org.apache.flink.streaming.api.functions.windowing.ProcessWindowFunction;
+import org.bptlab.cepta.models.events.correlatedEvents.CountOfTrainsAtStationEventOuterClass.*;
+import org.bptlab.cepta.models.events.train.LiveTrainDataOuterClass.LiveTrainData;
+import org.apache.flink.util.Collector;
+import org.apache.flink.api.java.functions.KeySelector;
+import org.apache.flink.streaming.api.windowing.time.Time;
+import org.apache.flink.streaming.api.windowing.assigners.SlidingEventTimeWindows;
+import org.apache.flink.streaming.api.windowing.windows.TimeWindow;
+
+/**
+ * This class implements a function that takes in an LiveTrainStream and return a stream of Tuple2<Long, Integer>
+ * where each tuple describes a station and the count of trains inside that station in an one hour
+ * window.
+ *
+ * This Window slides every 15 minutes. So each train will potentially be counted multiple times
+ * in different windows.
+ */
+public class CountOfTrainsAtStationFunction {
+
+    public static DataStream<CountOfTrainsAtStationEvent> countOfTrainsAtStation(DataStream<LiveTrainData> inputStream) {
+        DataStream<CountOfTrainsAtStationEvent> resultStream = inputStream
+        .keyBy(
+            new KeySelector<LiveTrainData, Long>(){
+                public Long getKey(LiveTrainData event){
+                    return event.getStationId();
+                }
+            }
+        )
+        .window(SlidingEventTimeWindows.of(Time.hours(1), Time.minutes(15)))
```

Comment:
Maybe define the times as final static vars. E.g. WINDOW_SIZE = 1, SLIDING_INTERVAL = 15 with short comment regarding the time unit (hours/minutes)

Reply:
I think thats a cool idea. How about defining
`private final static Time WINDOW_SIZE = Time.hours(1);`
in our class and then  `.of(WINDOW_SIZE, SLIDING_INTERVAL)`? 
I think the Time.hours is descriptive enough so we wouldnt need a comment

Reply:
Sounds good :)